features/repository/show/date_select.feature

```diff
@@ -18,7 +18,7 @@ Feature: Date Select
     Then I should see "Retrieve the closest processing information from:"
     When I set the select field "day" as "1"
     And I set the select field "month" as "1"
-    And I set the select field "year" as "2013"
```

Comment:
So in 2017 this test is broken? I believe this has another source. I'll comment there.